erlang-25.yaml

```diff
@@ -10,7 +11,7 @@ package:
       # mnesia depends on the ca-certificates bundle
       - ca-certificates-bundle
     provides:
-      - erlang=25.3.2.2
+      - erlang=25.999.0
```

Comment:
Do you know why we're setting 25.999.0 here instead of package.version? I can see the existing erlang26.yaml package does the same but wasn't clear

Reply:
I am not quite sure why we do this instead of full version. It has something to do with apk and how it resolves package conflicts. But I have seen this when we haver multiple version of a package for example [consul-1.15](https://github.com/wolfi-dev/os/blob/4a4136721f9f19d20ea7dc778648ed500789c4b2/consul-1.15.yaml#L10) and [consul-1.16 ](https://github.com/wolfi-dev/os/blob/4a4136721f9f19d20ea7dc778648ed500789c4b2/consul-1.16.yaml#L10). and also saw these in mariadb and nodejs etc.

here i just saw other erlang packages and done the same. Definitely an interesting question though, will look into it or ask someone.

Reply:
I am gonna fix this and also the version 26 now